packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Btc/Wallets/selectors.js

```diff
@@ -1,7 +1,18 @@
 import { map } from 'ramda'
+
 import { selectors } from 'data'
+import { Types } from 'blockchain-wallet-v4'
 
 export const getData = state => {
-  const accounts = map(x => ({ label: x.label, value: x }))
-  return selectors.core.common.bitcoin.getAccountsBalances(state).map(accounts)
+  const defaultId = Types.HDWallet.selectDefaultAccountIdx(Types.Wallet.selectHdWallets(state.walletPath.wallet).get(0))
+  const wallets = map(x => ({
+    label: x.label,
+    index: x.index,
+    archived: x.archived,
+    default: defaultId === x.index,
+    balance: x.info.final_balance,
+    xpub: x.xpub
+  }))
+
+  return selectors.core.common.bitcoin.getActiveHDAccounts(state).map(wallets)
```

Comment:
Is there a getHDAccounts selector instead? Noticed that if I archive an account there is no way to unarchive it.